docs/stdlib/constraints.rst

```diff
@@ -121,18 +110,17 @@ Constraints
 
     :index: regex regexp regular
 
-    Specifies that the string representation of the subject must match a
-    regexp.
-
-    Example:
+    Specifies the allowed representation of a type by matching a given
```

Comment:
```suggestion
    Limits to values with string representations matching a regular expression.
```

This gets us down to one line and better reflects the constraint. A value of a type isn't really a representation of a type but is more like an instance of it.